[enterprise-4.5] BZ-1843158: Backport Removing cloud provider credent… (#32152)
openshift/openshift-docs: modules/manually-removing-cloud-creds.adoc

// Module included in the following assemblies:
//
// * installing/installing_aws/installing-aws-account.adoc

[id="manually-removing-cloud-creds_{context}"]
= Removing cloud provider credentials

> After installing an {product-title} cluster on Amazon Web Services (AWS), you can remove the administrator-level credential secret from the `kube-system` namespace in the cluster. The administrator-level credential is not required unless something that requires an administrator-level credential needs to be changed, for instance during an upgrade.

Reviewer: I think the second sentence can be tightened a little. Maybe:

```suggestion
After installing an {product-title} cluster on Amazon Web Services (AWS), you can remove the administrator-level credential secret from the `kube-system` namespace in the cluster. The administrator-level credential is required only during changes that require its elevated permissions, such as upgrades.
```
